Sources/Sentry/include/SentryFormatter.h

```diff
@@ -0,0 +1,36 @@
+#import <Foundation/Foundation.h>
+
+// 2 for the 0x prefix, plus 16 for the hex value, plus 1 for the null terminator
```

Comment:
`m`: Please document here somewhere how much faster `snprintf` is than `[NSString stringWithFormat:@"0x%016llx", [value unsignedLongLongValue]];`, so we know why we did this and don't change it back in the future.

Reply:
I'll add something like the PR description here!

Reply:
added here: https://github.com/getsentry/sentry-cocoa/pull/2863/commits/8a345884c4e1a32453fc7dd832e5e576d9c66cc1